recipe/meta.yaml

```diff
@@ -1,70 +1,77 @@
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set version = "0.8.3.1" %}
+{% set sha256 = "4b0f5a5b4828915d973d3cbb86372aea9207e7b60fed912045d69242a74407f6" %}
+
 package:
-    name: ulmo
-    version: 0.8.2
+  name: ulmo
+  version: {{ 0.8.3.1 }}
```

Comment:
Been there, done that.

@dharhas it is fair game to self-merge updates like this when you are a maintainer. So once the CIs are green feel free to merge this. Thanks again for the new release and recipe update!